src/TinyGsmClientSIM7600.h

```diff
@@ -117,12 +128,86 @@ class TinyGsmSim7600 : public TinyGsmModem<TinyGsmSim7600>,
       stop();
       TINY_GSM_YIELD();
       rx.clear();
-      sock_connected = at->modemConnect(host, port, mux, false, timeout_s);
+      sock_connected = at->modemConnect(host, port, mux, SSLVersion::NO_SSL,
+                                        timeout_s);
       return sock_connected;
     }
     TINY_GSM_CLIENT_CONNECT_OVERRIDES
 
-    void stop(uint32_t maxWaitMs) {
+    virtual size_t modemGetAvailable(uint8_t mux) {
```

Comment:
The `mux` is already available as `this->mux`, so no need to pass it here; same applies to the other functions here.

Moreover, these three functions can be `protected` since they are only ever used by `TinyGsmSim7600`.